test/HasPredicateScanTest.cpp

```diff
@@ -13,7 +13,7 @@
 class DummyOperation : public Operation {
  public:
   DummyOperation(QueryExecutionContext* ctx) : Operation(ctx) {}
-  virtual void computeResult(ResultTable* result) {
+  void computeResult(ResultTable* result) {
```

Comment:
I know it's just a test in a dummy, but why isn't this `virtual` + `override` like everywhere else?
Edit: at least `override`?